docs/glues/herbs2gql.md

```diff
@@ -1,245 +1,11 @@
 ---
 id: herbs2gql
-title: GraphQL - Herbs2GQL
 sidebar_label: GraphQL
 slug: /glues/herbs2gql
+hide_title: true
 ---
 
-Creates GraphQL types (queries, mutations, etc.) based on herbs [entities](/docs/entity/getting-started) and [usecases](/docs/usecase/getting-started), based on [Apollo](https://www.apollographql.com/) GraphQL.
+import ReadMeDoc from '../../src/components/ReadMeDoc'
 
+<ReadMeDoc docURL='https://raw.githubusercontent.com/herbsjs/herbs2gql/master/README.md'/>
```

Comment:
I think this should be inside a env file

Reply:
I wasn´t able to put it inside the md file, but I put the base URL inside the env file and oly pass the endpoint at the md file.